pkg/machine/e2e/qemu_command_test.go

```diff
@@ -0,0 +1,78 @@
+package e2e_test
+
+import (
+	"fmt"
+
+	"github.com/containers/podman/v4/pkg/machine"
+	"github.com/containers/podman/v4/pkg/machine/qemu"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("run basic podman commands", func() {
+	var (
+		mb      *machineTestBuilder
+		testDir string
+	)
+
+	BeforeEach(func() {
+		testDir, mb = setup()
+	})
+	AfterEach(func() {
+		teardown(originalHomeDir, testDir, mb)
+	})
+
+	It("QemuCmd builder setup", func() {
```

Comment:
This is not a e2e test, this is an unit test.
As such this should be moved into the qemu dir and turned into a regual unit test, first these tests here run on real hardware so time is much more expensive and the setup here is just wasted time.